### PR TITLE
apiv2 - API support for top authors metrics

### PR DESCRIPTION
**Pull request by morucci**
related to #468

### CODE REVIEW

File: haskell/src/Monocle/Backend/Queries.hs [hunk 1]
```diff
@@ -300,7 +300,7 @@ getQueryFromSL query =
 getSimpleTR :: BH.TermsResult -> TermResult
 getSimpleTR tr = TermResult (getTermKey tr) (BH.termsDocCount tr)
 
-data TermResult = TermResult {term :: Text, count :: Int} deriving (Show, Eq)
+data TermResult = TermResult {tRterm :: Text, tRcount :: Int} deriving (Show, Eq)
```
Comment (TristanCacqueray) [inline haskell/src/Monocle/Backend/Queries.hs:303]: We usually keep the prefix lowercase and make the actual record field PascalCase.
```suggestion
data TermResult = TermResult {trTerm :: Text, trCount :: Int} deriving (Show, Eq)
```

Comment (TristanCacqueray) [reply haskell/src/Monocle/Backend/Queries.hs:303]: Though, why not using the schema directly from SearchPB?

**Review by TristanCacqueray — APPROVED** [review]

File: haskell/src/Monocle/Backend/Queries.hs (hunk 1 above)
Comment (morucci) [reply haskell/src/Monocle/Backend/Queries.hs]: If you don't mind I'll prefer to keep a segmentation here

**Comment by morucci** [comment]
This one is good to go

**Review by TristanCacqueray — APPROVED** [review]

**Comment by TristanCacqueray** [comment]
Would you mind if we merge https://github.com/change-metrics/monocle/pull/496 first, then let's rebase that branch.

**Comment by TristanCacqueray** [comment]
It seems like this used the wrong workflow, could we recheck?

**Pull request merged**